roles/backup-server/templates/usr/local/bin/run-backups.sh.j2

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -e
```

Comment:
Is there anything here preventing a cron executions from stacking up on each other, should a backup go long or get stuck in some way?